test/author_test.rb

```diff
@@ -0,0 +1,29 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/book'
+require './lib/author'
+
+class AuthorTest < MiniTest::Test
+
+  def setup
+    @charlotte_bronte = Author.new({first_name: "Charlotte", last_name: "Bronte"})
+    @book = Book.new({author_first_name: "Charlotte", author_last_name: "Bronte", title: "Jane Eyre", publication_date: "October 16, 1847"})
+    @book2 = Book.new({author_first_name: "Charlotte", author_last_name: "Bronte", title: "Villette", publication_date: "1853"})
+  end
+
+  def test_it_exists
+    assert_instance_of Author, @charlotte_bronte
+  end
+
+  def test_it_has_attributes
+    assert_equal "Charlotte Bronte", @charlotte_bronte.name
+    assert_equal [], @charlotte_bronte.books
+  end
+
+  def test_write
+    assert_equal @book, @charlotte_bronte.write("Jane Eyre", "October 16, 1847")
```

Comment:
Rather than creating books in the setup method, and creating a comparison method to verify the books being written in the `write` method, I would do something like this:

```ruby
jane_eyre = @charlotte_bronte.write("Jane Eyre", "October 16, 1847")
villette = @charlotte_bronte.write("Villette", "1853") 

assert_equal "Jane Eyre", jane_eyre.title
assert_equal "Charlotte Bronte", jane_eyre.author
assert_equal "1847", jane_eyre.publication_year
# this verifies that books are being created correctly ^^
assert_equal [jane_eyre, villette], @charlotte_bronte.books
```